src/chromo/constants.py

```diff
@@ -53,32 +55,6 @@
     5503,
 )
 
-# standard long-lived particles with life-time > 30 ps
-long_lived = (
-    13,
-    2112,
-    130,
-    211,
-    310,
-    321,
-    -3334,
-    -3322,
-    -3312,
-    -3222,
-    -3122,
-    -3112,
-    3112,
-    3122,
-    3222,
-    3312,
-    3322,
-    3334,
-    -321,
-    -211,
-    -2112,
-    -13,
-)
-
```

Comment:
You cannot remove this, it is public API. Please revert.

Reply:
I see you replaced this further below with a dynamic computation. I don't see the benefit, because I don't think we actually ever want to change the tau_stable threshold. So there is no need to make this dynamic as you did.

Reply:
I reverted it.